### DISCUSSION TITLE
How can I enable the NONPNP_HOOK_INT19 build-time option?

### DISCUSSION THREAD

**Discussion by anonymous5555522222** [ipxe/ipxe#1283] [Q&A] [answered]
How can I enable the NONPNP_HOOK_INT19 build-time option?

**Comment by mcb30** [comment]
> How can I enable the NONPNP_HOOK_INT19 build-time option?

More importantly, why do you want to enable this obscure option?

**Reply by anonymous5555522222** [reply]
The reason is that the NIC is going into a computer which motherboard does not have a "Boot from LAN" or similar option. Thanks in advance.

**Reply by mcb30** [reply, marked as answer]
Add the line

```
#define NONPNP_HOOK_INT19
```
to the (empty) file `config/local/general.h`

**Reply by anonymous5555522222** [reply]
> Add the line
> 
> ```
> #define NONPNP_HOOK_INT19
> ```
> 
> to the (empty) file `config/local/general.h`

Thank you.
